CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/GlyphRectangleBuffer.java

```diff
@@ -356,4 +357,14 @@ private boolean bufferedImagesEqual(BufferedImage img1, BufferedImage img2) {
         }
         return true;
     }
+
+    private synchronized Integer synchronizedAddHashcode(int hashCode, BufferedImage img, int extra, int w, int h) {
+        int value = memory.size();
+        Integer rectIndex = memory.putIfAbsent(hashCode, value);
```

Comment:
primitive `int`

Reply:
What happens if it returns a null?

Reply:
Right. If the return value can be null then never mind